src/standup/standup.service.ts

```diff
@@ -1,63 +1,69 @@
+import { AstraService, documentId } from '@cahllagerfeld/nestjs-astra';
 import { HttpException, HttpStatus, Injectable } from '@nestjs/common';
 import { StandupDTO } from './dto/standup.dto';
 import { Standup } from './interfaces/standup.interface';
+import { tap } from 'rxjs/operators';
 
 @Injectable()
 export class StandupService {
-  private standups: Array<Standup> = [];
+  constructor(private readonly astraService: AstraService) {}
 
-  create(body: StandupDTO): Standup {
+  create(body: StandupDTO) {
     const { discordUser, todayMessage, yesterdayMessage } = body;
 
     const newStandup: Standup = {
-      id: 123,
       yesterdayMessage: yesterdayMessage,
       todayMessage: todayMessage,
       discordUser: discordUser,
-      createdOn: new Date('2021-01-01T00:00:00.000Z'),
+      createdOn: new Date(Date.now()),
     };
 
-    this.standups.push(newStandup);
-
-    return newStandup;
+    // this.standups.push(newStandup);
+    return this.astraService.create<Standup>(newStandup).pipe(
+      tap((data: documentId) => {
```

Comment:
I think `tap` might be a bit risky here as the response might already have been returned. Would a `filter` work here?

Reply:
Another example for `tap`, would be to use it for `logging` where the action does not have any effect on the result

Reply:
I have to try it. I'm quite new to Rxjs and didnt even know about `filter`. 😅

Reply:
No worries, RxJS is massive, I am still learning about it all the time too